pages/common/ia.md

```diff
@@ -0,0 +1,24 @@
+# ia
+
+> Command-line tool to interact with archive.org.
+> More information: <https://archive.org/services/docs/api/internetarchive/cli.html>.
+
+- Configure ia to get your API keys; certain functions won't work without this step:
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
- Configure `ia` with API keys (some functions won't work without this step):
```